collection-scripts/targeted_crs

```diff
@@ -119,14 +119,25 @@ if [ ! -z "${vm_resources}" ]; then
 
         # Gather VMs PersistentVolumeClaims
         for pvc_name in $(echo $vm_data | jq -r '.spec.template.spec.volumes[] .persistentVolumeClaim.claimName'); do
-          # In order to gather the scratch PVCs in warm migrations
-          for pvc in $(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]'); do
-            if [ $(echo $pvc | jq -r '.spec.volumeName') != $pvc_name ]; then
+          pvcs=$(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]')
+          pop_volume_name=""
+          for pvc in ${pvcs[@]}; do
```

Comment:
something is messy here.
If I use it as is - jq breaks:
```parse error: Unfinished string at EOF at line 2, column 0
-bash: [: =: unary operator expected
parse error: Invalid numeric literal at line 1, column 9
-bash: [: =: unary operator expected
```

If I change it to `"${pvcs[@]}"`, jq will be able to read it. but it will be a single string, therefore resulting in too many arguments to the `if` afterwards...

Reply:
that's good but then `pvc` is not the right term to represent what we got, how about `pvc_info`?